modules/ScreenNameLocalFormatter/src/main/java/com/liferay/formatter/keys/NewFormatterKeys.java

```diff
@@ -0,0 +1,7 @@
+package com.liferay.formatter.keys;
+
+public interface NewFormatterKeys {
+
+	public static final String USERS_SCREEN_NAME_COMPANY_EMAIL =
```

Comment:
Formatting is incorrect.
Run `gw formatSource` to fix some of the issues.
Then run `gw checkSource` to validate our code is properly formatted.
If `checkSource` fails, you'll have to format the code manually

Reply:
Both commands were used and formatting issues are fixed now.